scripts/main/album.js

```diff
@@ -937,6 +937,131 @@ album.setPublic = function (albumID, e) {
 	});
 };
 
+album.shareUsers = function (albumID, e) {
+	var addShare = function (listIds) {
+		var params = {
+			albumIDs: albumID,
+			UserIDs: listIds.join(","),
+		};
+		api.post("Sharing::Add", params, (data) => {
+			if (data !== true) {
+				loadingBar.show("error", data.description);
+				lychee.error(null, params, data);
+			} else {
+				loadingBar.show("success", "Sharing updated!");
+			}
+		});
+	};
+
+	var removeShare = function (listIds) {
+		var params = { ShareIDs: listIds.join(",") };
+		api.post("Sharing::Delete", params, function (data) {
+			if (data !== true) {
+				loadingBar.show("error", data.description);
+				lychee.error(null, params, data);
+			}
+		});
+	};
+
+	if (!basicModal.visible()) {
+		let msg = `<form id="sharing_people_form">
+			<p>${lychee.locale["WAIT_FETCH_DATA"]}</p>
```

Comment:
When testing, I did see the message. It was only for a few seconds, but still.

I think it's better to have a message saying that it's normal if the user don't see anything yet, rather than displaying anything and just waiting to have the data from the API.

Reply:
Your server must be far away :smiley:.